components/css/newsletter-text-only/_contentPage.scss

```diff
@@ -0,0 +1,173 @@
+$white:#ffffff;
+$black:#000000;
+$light-gray:#aaaaaa;
+$dark-gray:#2e2e2e;
+$gold:#c28e0e;
+
+.intro {
+	background-color: #c28e0e;
+	padding: 1.5em;
+
+
+	&__title {	
+	font-size: 2em;
+	line-height: 1.3em;
+	color: $black;
+	text-align: center;
+	text-transform:uppercase;
+	padding: .5em 0;
+	}
+	
+	&__issue {
+	font-size: 1em;
+	line-height: 1em;
+	color: $black;
+	text-align: center;
+	padding: 0;
+	}
+	
+	&__issue::after {
+	content:" ";
+	display: block;
+	width: 30%;
+	height: 3px;
+	margin: .3em auto !important;
+	background-color: $white;	
+	}
+	
+	&__description {
+	font-size: .9em;
+	line-height: 1.4em;
+	color: $black;
+	text-align: center;
+	padding: 1.5em 0 0 0;
+	}
+}
+
+
+.section {
+	background-color: $dark-gray;
+	padding: 1.5em;
+	
+	&__heading {
+	font-family:Impact,"Helvetica Neue",Helvetica,sans-serif;
+	color: $white;
+	font-size: 1.6em;
+	line-height: 1.2em;
+	text-transform:uppercase;
+	padding: .5em 0 0 0;
+	}
+	
+	&__heading::after {
+	content:" ";
+	display: block;
+	width: 20%;
+	height: 3px;
+	margin: .2em 0 0 0 !important;
+	background-color: $gold;	
+	}
+}
+
+.story {
+	margin: 0;
+	padding: 0;
+		
+	&__title {
+	color: $gold;
+	font-size: 1.4em;
+	line-height: 1.3em;
+	text-align: left;
+	padding: 1em 0 .4em 0;
+	}
+	
+	&__subtitle {
+	color: $light-gray;
+	font-size: 1em;
+	line-height: 1.5em;
+	text-align: left;
+	padding: 0 0 .8em 0;
+	}
+	
+	&__teaser {
+	color: $white;
+	font-size: .9em;
+	line-height: 1.5em;
+	text-align: left;
+	padding: 0 0 .5em 0;
+	}
+}	
+
+
+.list {
+	list-style-type: none;
+	width: 330px;
+	padding: 1.5em 1.5em 0 .5em;
+	
+
+	li {
+	color: $gold;
+	line-height: 1.5em;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	}
+
+	li span:first-child {
+	flex-grow: 0;
+	flex-shrink: 0;
+	flex-basis: 3em;
+	}
+
+	li span:last-child {
+	flex-grow: 1;
+	font-size: .9em;
+	line-height: 1.4em;
+	}
+
+	.fa-2x {
+	width: 3em;
+	color: $black;
+	margin: 0 0 0 -1em;
+	}
+
+	.fa-newspaper-o {
+	color: $light-gray;
+	}
+	
+	&--light {
+		background-color:$white;	
+	}
+}
+
+.button {
+	margin: 1em 0 0 0 !important;
+}
+
+.cta-archive {
+	width: 100%;
+	padding: 1em;
+	background: repeating-linear-gradient(60deg,#707070,#707070 6px,#777777 6px,#777777 8px);
```

Comment:
Perhaps we should add another mixin that contains a function that accepts 2 arguments:
- The hex value of the first color used in the stripes
- The hex value of the second color used in the stripes

It will return `repeating-linear-gradient(60deg,[first color here],[first color here] 6px,[second color here] 6px,[second color here] 8px);`. That way, we only need to feed colors to the stripes and we ensure consistent spacing of stripes throughout the templates.

Reply:
I like the idea. Should we account for any variations in degrees of line thickness?

Reply:
Maybe not since we want to have a standard look for stripes. Although, we could provide the option to supply the line sizes but provide a default if they aren't provided.

Reply:
Could also have just a few styles like "thin, "medium," or "thick"

Reply:
So would it be "background: repeating-linear-gradient(60deg,[#707070],[#707070] 6px,[#777777] 6px,[#777777] 8px);"? 

I have an error message when I have it like this though.

Reply:
resolved